addons/escoria-core/game/escoria.gd

```diff
@@ -315,7 +315,7 @@ func _ev_left_click_on_item(obj, event, default_action = false):
 			"arrived"
 		)
 		
-		self.logger.info("Context arrived: %s" % var2str(context))
+		self.logger.info("Context arrived: %s" % context)
```

Comment:
Ah, does context return a readable representation when called in a string context? Nice.

Reply:
In fact, this was causing a crash due to an infinite loop, related to `_get_property_list()`. Not sure why. But yes, `Dictionary` is readable directly when printed.